sklearn/pipeline.py

```diff
@@ -184,6 +184,26 @@ def predict(self, X):
         return self.steps[-1][-1].predict(Xt)
 
     @if_delegate_has_method(delegate='_final_estimator')
+    def fit_predict(self, X, y=None, **fit_params):
+        """Exposes fit_predict on pipelines.
```

Comment:
Agreed. what about `Applies fit_predict of last step in pipeline after transforms.` as the summary?